samples/python-azurefunction/README.md

```diff
@@ -15,6 +15,9 @@ This sample requires you to have the following installed on your machine:
 - [Set up Python Environment in Visual Studio Code](https://code.visualstudio.com/docs/python/python-tutorial)
 - [Install .NET Core SDK](https://dotnet.microsoft.com/download), used for install Dapr Extension for non .NET language
 - [Run Kafka Docker Container Locally](https://github.com/dapr/quickstarts/tree/master/bindings). The required Kafka files is located in `sample\dapr-kafka` directory.
+```
+docker-compose -f docker-compose-single-kafka.yml up
```

Comment:
Is this file already available here, or do they need to download from somewhere?

Reply:
This file available in [samples\dapr-kafka](https://github.com/Azure/azure-functions-dapr-extension/blob/master/samples/dapr-kafka/docker-compose-single-kafka.yml) folder,